### ISSUE TITLE
Wikipedia - issues 

### ISSUE THREAD

**Issue by darkred** [tophf/mpiv#11]
I've just noticed the following two issues with any wikipedia page (using latest MPIV with Tampermonkey Beta/Violentmonkey in fresh Chrome profile):

1. There's no popup for any image I mouseover with any wikipedia page.

   Example URL: https://en.wikipedia.org/wiki/Concert
   I use the following custom rule from the repository, changing "wikimedia" to wikipedia in its `r`:

   ```json
   {"r": "wikipedia\\.org/wiki/File",
    "q": ".fullImageLink a",
    "follow": true
   }
   ```
   Also, in the log (having MPIV settings open while mouseover) 
   it only shows as matched rule the following built-in one:
`{"r": "/^[^?]+?\.(bmp|jpe?g?|gif|mp4|png|svg|webm)($|\?)/i"`
but there's no popup at all, as I said.

   (for reference: in the original MPIV, the popup still works ok in wikipedia pages by default, i.e. without adding a custom rule)

&nbsp;

2. When I open MPIV settings in the above wikipedia URL and press "Install rule from repository", 
the relevant entry "Wikipedia (ALL lists)" in the rule list doesn't become auto-selected. 
    I also tried https://en.wikipedia.org/wiki/Main_Page and it still doesn't becomes auto-selected.

    (for reference: in the original MPIV the relevent entry becomes auto-selected )

---

The following "issue" is minor, but it often causes confusion to me : 

in the original MPIV, when you: 
- clicked on a rule in https://w9p.co/userscripts/mpiv/more_host_rules.html , 
- (in Settings) pressed "Install rule from repository", selected a rule from the list and pressed "Install",

the rule wasn't stored in Settings **unless you pressed the "Ok" button to confirm the rule save**.

With this fork, in both cases (about the latter, now there's no "Install" button - you doubleclick/press Enter a rule from the list), the rule becomes immediately saved the moment it's displayed in Settings, i.e it doesn't matter whether you press Save/Apply afterwards. 

**Could you please change it so that the Save/Apply buttons act as "confirmation", like in the original MPIV?**

**Comment by darkred** [comment]
Update regarding the 1st issue:

I think that it doesn't have to do especially with wikipedia.org  :

I've just tried e.g. the following rule (made for https://blog.hubspot.com/sales/email-multiple-inboxes )

```json
{"d": "blog.hubspot.com",
 "r": ".*\\.(jpe?|pn)g.*",
 "s": "/\\?width=.*//"
}
```

and it also doesn't work (it works ok in original MPIV).

In the log it only displays the built-in mentioned above `{"r": "/^[^?]+?\.(bmp|jpe?g?|gif|mp4|png|svg|webm)($|\?)/i"` as matched rule, like before.

**Comment by tophf** [comment]
>There's no popup for any image I mouseover with any wikipedia page

Can't repro. Works for me regardless of your custom rule. I've even removed all my custom rules by importing `{}`

>the rule becomes immediately saved

Not immediately, but on pressing the `cancel` button which is a relatively recent bug. Fixed now.

>the following rule (made for https://blog.hubspot.com

Works for me. Maybe you've changed some option in MPIV like `if larger` or maybe you have a custom rule that interferes.

**Comment by tophf** [comment]
I wonder if you were still using the broken version (e053fe2e) which I fixed (3a5e0595) an hour before you opened the issue.

**Comment by darkred** [comment]
> I wonder if you were still using the broken version ([e053fe2](https://github.com/tophf/mpiv/commit/e053fe2e810eac2cd6f74ed9c7b7042aada70851)) which I fixed ([3a5e059](https://github.com/tophf/mpiv/commit/3a5e05954656eee4b8000900f733d7925c5c81c9)) an hour before you opened the issue.

Yes, I had stayed to that broken version, while I was making this report.

Thanks a lot for the quick response and for fixing the other two issues.
